<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSetting</class>
 <widget class="QDialog" name="FrmSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferencias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);
background-color: rgb(33, 48, 58);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 62, 62);
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(250, 128, 28, 255), stop:0.98 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 0));

color: rgb(0, 0, 0);


  border: 2px solid #000000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Preferencias</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 62, 62);
color: rgb(255, 255, 255);
  border: 2px solid #000000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/menu/prefer.png</normaloff>:/general/iconos/menu/prefer.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pisador</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/menu/prefer.png</normaloff>:/general/iconos/menu/prefer.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 62, 62);
color: rgb(255, 255, 255);

  border: 1px solid #000000;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>481</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Solapar pistas a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Solapar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>30</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Segundos del final</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Descartar pistas menores de</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Descartar">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Segundos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>481</width>
           <height>91</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Tiempo de Fundido</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Fundir">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Segundos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="FundirSolapar">
                <property name="text">
                 <string>Fundidos al Solapar</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="FundirParar">
                <property name="text">
                 <string>Fundido al Parar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>481</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="Dispositivo"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>481</width>
           <height>121</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <widget class="QCheckBox" name="Detector">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>201</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Activar Detector</string>
          </property>
         </widget>
         <widget class="QVUMeter" name="VuNivel">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>10</y>
            <width>50</width>
            <height>101</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>VU Meter</string>
          </property>
          <property name="whatsThis">
           <string>VU Meter</string>
          </property>
          <property name="colorBg">
           <color>
            <red>33</red>
            <green>48</green>
            <blue>58</blue>
           </color>
          </property>
          <property name="colorValue">
           <color>
            <red>33</red>
            <green>48</green>
            <blue>58</blue>
           </color>
          </property>
          <property name="colorLow">
           <color>
            <red>79</red>
            <green>145</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="colorHigh">
           <color>
            <red>79</red>
            <green>145</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="maxValue">
           <double>120.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="DetectorNivel">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>50</y>
            <width>62</width>
            <height>22</height>
           </rect>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Dispositivo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>66</y>
           <width>111</width>
           <height>15</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Superposición</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>145</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Fundidos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>255</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Detector de finales</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QFrame" name="frame_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>481</width>
           <height>46</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Variación de</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="Transicion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="value">
               <number>500</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Milisegundos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>481</width>
           <height>121</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);</string>
         </property>
         <widget class="QVUMeter" name="VuNivel2_3">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>10</y>
            <width>50</width>
            <height>101</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>VU Meter</string>
          </property>
          <property name="whatsThis">
           <string>VU Meter</string>
          </property>
          <property name="colorBg">
           <color>
            <red>33</red>
            <green>48</green>
            <blue>58</blue>
           </color>
          </property>
          <property name="colorValue">
           <color>
            <red>33</red>
            <green>48</green>
            <blue>58</blue>
           </color>
          </property>
          <property name="colorLow">
           <color>
            <red>79</red>
            <green>145</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="colorHigh">
           <color>
            <red>79</red>
            <green>145</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="leftValue">
           <double>0.150000000000000</double>
          </property>
          <property name="rightValue">
           <double>0.150000000000000</double>
          </property>
          <property name="maxValue">
           <double>1.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="Locucion">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>50</y>
            <width>62</width>
            <height>22</height>
           </rect>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.150000000000000</double>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Transición</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>75</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Locución</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSalir">
       <property name="styleSheet">
        <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVUMeter</class>
   <extends>QWidget</extends>
   <header>qvumeter.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="radit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>label_2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DetectorNivel</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VuNivel</receiver>
   <slot>setLeftValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DetectorNivel</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VuNivel</receiver>
   <slot>setRightValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Locucion</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VuNivel2_3</receiver>
   <slot>setLeftValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Locucion</sender>
   <signal>valueChanged(double)</signal>
   <receiver>VuNivel2_3</receiver>
   <slot>setRightValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ClickSalir()</slot>
 </slots>
</ui>
